core/src/main/kotlin/org/jetbrains/research/testspark/core/monitor/ErrorMonitor.kt

```diff
@@ -0,0 +1,25 @@
+package org.jetbrains.research.testspark.core.monitor
+
+interface ErrorMonitor {
+    fun notifyErrorOccurrence(): Boolean
+    fun isErrorOccurred(): Boolean
```

Comment:
On more comment about the name of this method: "error is occurred" is grammatically incorrect, hence `isErrorOccurred` contains a grammatical error either. "error [has] occurred" is correct. 

So, I suggest to rename it to either `hasErrorOccurred` or `errorOccurred`.

Reply:
Fixed by c0c42c47cf616ca56a6321d2ba6b56498984b5ed